sycl/test/Unit/lit.cfg.py

```diff
@@ -39,11 +41,7 @@
     # force it load the default configuration.
     config.environment['SYCL_CONFIG_FILE_NAME'] = "null.cfg"
 
-if 'SYCL_DEVICELIB_NO_FALLBACK' in os.environ:
-    config.environment['SYCL_DEVICELIB_NO_FALLBACK'] = os.environ['SYCL_DEVICELIB_NO_FALLBACK']
-else:
-    # Disable device library fallback for unit tests by default.
-    config.environment['SYCL_DEVICELIB_NO_FALLBACK'] = "1"
+llvm_config.with_system_environment(['SYCL_DEVICELIB_NO_FALLBACK'])
```

Comment:
I'm afraid I'm not qualified to review this piece.

Reply:
I've simplified this code to simply remove `else` branch in 697795ac0f13fec6dd730e00cc7eedfd0251cac0